package.json

```diff
@@ -108,7 +119,8 @@
     "mocha-only-detector": "1.0.0",
     "prettier": "^1.14.3",
     "rimraf": "^2.6.2",
-    "rsvp": "^4.8.4"
+    "rsvp": "^4.8.4",
+    "typescript": "~3.1.3"
```

Comment:
TS does not follow SemVer, so floating on minor releases has the potential to troll consumers.

Reply:
I just got my gmail that was connected to my drefi  account back back so trying to fix the issue